<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTranferListItem</class>
 <widget class="QWidget" name="FileTranferListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelMessage">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>正在等待下载...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSpeed">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProgress">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QLabel" name="labelSourceFile">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelDestinationFile">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonPause">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>101</horstretch>
           <verstretch>31</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>暂停(&amp;S)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonContinue">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>101</horstretch>
           <verstretch>31</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>继续(&amp;O)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>101</horstretch>
           <verstretch>31</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>取消(&amp;C)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRetry">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>重试(&amp;R)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
